src/main/java/chanceCubes/rewards/giantRewards/PotionsReward.java

```diff
@@ -54,7 +53,7 @@ public void update()
 				{
 					MobEffectInstance potionEffect = RewardsUtil.getRandomPotionEffectInstance();
 					pot = new ThrownPotion(level, player);
-					pot.setItem(PotionUtils.setCustomEffects(new ItemStack(Items.SPLASH_POTION), List.of(potionEffect)));
+//					pot.setItem(PotionUtils.setCustomEffects(new ItemStack(Items.SPLASH_POTION), List.of(potionEffect))); TODO: Fix potions reward!
```

Comment:
What's changed with the game here?

Reply:
Whoops... I forgot I commented it out. I had already fixed it in other locations.